app/assets/javascripts/spina/projects/admin/sorting.coffee

```diff
@@ -0,0 +1,26 @@
+set_positions = ->
```

Comment:
I think you can loose this file completely. The change I have suggested to the list will hook into the pages sorting JS already in Spina.